doc/whatsnew/fragments/3585.false_negative

```diff
@@ -0,0 +1,7 @@
+`invalid-name` now distinguishes module-level names that are assigned only once
```

Comment:
This should probably be advertised as a breaking change for 4.0, it make sense to do it this way but pylint have been doing it a certain way for so long that "module level as constant" became a *convention* of a kind. (``LOGGER`` vs ``logger`` was not settled last time I checked, admittedly a long time ago).